src/blue-theme.scss

```diff
@@ -0,0 +1,21 @@
+@charset "utf-8";
+
+// Set brand colors
+$democracy-blue: #2e4272;
+
+// Update Bulma's global variables
+$primary: $democracy-blue;
+$tooltip-background-color: $democracy-blue;
+
+@import "~bulma/bulma";
+@import "~bulma-tooltip";
+
+[class*=" lavinia-logo"],
+[class^="lavinia-logo"] {
+  letter-spacing:2px;
+  stroke-width:.26458px;
+  stroke: findColorInvert($primary);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "findColorInvert" to be "findcolorinvert" (function-name-case)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5535655)